examples/7_extended_write_serial.cpp

```diff
@@ -9,7 +9,7 @@ int main()
 
     {
         auto f =
-            io::Series("working/directory/2D_simData.h5", io::Access::CREATE);
+            io::Series("working/directory/2D_simData.bp", io::Access::CREATE);
```

Comment:
@franzpoeschel do you want to have this change in the release?

If so, we need to runtime guard this because we have CI tests that only have h5 installed (intentionally). I would also move this in a separate PR, this PR is purely documentation :)